.travis.yml

```diff
@@ -77,9 +76,8 @@ matrix:
       before_install:
         - eval "$(curl -sL https://gist.githubusercontent.com/kylef/5c0475ff02b7c7671d2a/raw/9f442512a46d7a2af7b850d65a7e9bd31edfb09b/swiftenv-install.sh)"
       script:
-        - mv .Package.test.swift Package.swift
         - swift build
-        - swift test
+        - SWIFTPM_TEST_ReactiveSwift=YES swift test
```

Comment:
I'd rather run `swift build` without the test dependencies so I added the env variable to explicitly `swift test`.